File: frontend/src/app/core/helpers/signalr/SignalRFlowMessageButtonResponse.ts
import FlowMessageButtonId from "../../entities/flow-messages/flow-message-buttons/keys/FlowMessageButtonId";
import FlowMessageId from "../../entities/flow-messages/flow-message/keys/FlowMessageId";

export interface SignalRFlowMessageButtonResponse {
  id: FlowMessageButtonId;
  displayText: string;
  next: FlowMessageId;
}

File: backend/SprintBusiness/SprintBuisness.Contracts/Whatsapp/Dtos/MessageResponse/SignalRFlowMessageButtonResponse.cs
using SprintBusiness.Domain.Flows.FlowMessageButtons.Keys;
using SprintBusiness.Domain.Flows.FlowMessages.Keys;

namespace SprintBuisness.Contracts.Whatsapp.Dtos.MessageResponse
{
    public class SignalRFlowMessageButtonResponse
    {
        public required FlowMessageButtonId Id { get; set; }
        public required string DisplayText { get; set; }
        public required FlowMessageId Next { get; set; }
    }
}
